snapcraft.yaml

```diff
@@ -215,4 +219,19 @@ parts:
       rm -rf usr/share/man
 
       find . -type d -empty -delete
-      $CRAFT_PROJECT_DIR/strip_binaries.py $CRAFT_PRIME
+
+      DEBUG_BUILDID=usr/lib/debug/.build-id
+      mkdir -p $DEBUG_BUILDID
+
+      $CRAFT_PROJECT_DIR/strip_binaries.py $CRAFT_PRIME $DEBUG_BUILDID
+
+      _VERSION=$SNAPCRAFT_PROJECT_VERSION
+      if [ "${_VERSION}" = "git" ]; then
+        pip3 install craft-parts
+        _VERSION=$(python3 -c "from craft_parts.sources.git_source import GitSource; print(GitSource.generate_version(part_src_dir='$CRAFT_PROJECT_DIR'))")
+      fi
+      SYMBOLS_ARCHIVE=$CRAFT_PROJECT_NAME_${_VERSION}_$SNAP_ARCH.debug
+      echo Compressing symbols
+      zip -r9 $CRAFT_PROJECT_DIR/$SYMBOLS_ARCHIVE $DEBUG_BUILDID
+      echo Symbols compressed
+      rm -fr $DEBUG_BUILDID
```

Comment:
This part seems mostly copy/paste of what I did, even though I am not proud of this, it serves its purpose